.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           shared-key: "shared"
 
       - name: Save sccache
-        uses: actions/cache@v3
+        uses: pat-s/always-upload-cache@v3
```

Comment:
what is the benefit here?

Reply:
By default, the actions/cache@v3 only uploads the cache if the run was successful. Unfortunately, the setting to change this is not exported. That's why I exchanged it with this other one which is basically the same except that it uploads the cache regardless of the overall outcome.